### DISCUSSION TITLE
Support of UI Automation for Theia based Application

### DISCUSSION THREAD

**Discussion by swathisuja** [redhat-developer/vscode-extension-tester#1474] [Ideas]
Support of UI Automation for Theia based Application 

"Does Extester support Theia/Electron? If not, whether the support can be added and when? "

**Comment by djelinek** [comment]
Hello, there is a related issue reported - https://github.com/redhat-developer/vscode-extension-tester/issues/1344

it is on the roadmap for upcoming weeks/months.. Currently there is a priority to resolve bugs first

**Reply by swathisuja** [reply]
We are in the process of migrating vs code extensions to theia based standalone application with all extensions. Wanted to know if Extester extends support to validate theia/electron based standalone application UI test  ?

**Comment by swathisuja** [comment]
@djelinek HI 

Would like to the updates on request for theia based application UI automation using Extester. Kindly share information.

**Reply by djelinek** [reply]
hello, I am sorry I had a time off.

 At the moment I am not sure. The possibility is here, but at the moment.. there will be a lot of work to make it working properly. First of all it needs to be checked how much compatible would be VS Code locators and Theia IDE locators.. and so on..

If you are interested in.. please feel free to raise a feature request, but it will require time and I am afraid I do not have capacity to work on it without broader community seeking for this (but, I will at least invest time to investigate what would it cost).
